mentors/python/l.json

```diff
@@ -31,4 +31,12 @@
     "avatar_url": null,
     "bio": "I love python and is the main programming language that I use in everyday, it's easy and beautiful language. I want to help everyone write more pythonic codes!"
   }
+  {
+  "github_username": "LorneHyde",
+  "name": "Lorne Hyde",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "My passion for coding began with python, and it led me to choosing to study Computer Science at university! I hope to share my love of python with others."
+}
```

Comment:
The lines I have hi-lighted need to each be indented 2 more spaces.  The formatting of your bio information should appear like the entries that precede yours.